src/Razor/src/Microsoft.AspNetCore.Razor.VSCode.BlazorWasmCompanionExtension/src/extension.ts

```diff
@@ -20,15 +20,8 @@ export function activate(context: vscode.ExtensionContext) {
             const debugProxyLocalPath = `${context.extensionPath}/BlazorDebugProxy/BrowserDebugHost.dll`;
             const spawnedProxyArgs = [debugProxyLocalPath , '--DevToolsUrl', debuggingHost];
 
-            let dotnet = "dotnet";
-            // The vscode.env.remoteName property is set when connected to
-            // any kind of remote workspace. See
-            // https://code.visualstudio.com/api/advanced-topics/remote-extensions#varying-behaviors-when-running-remotely-or-in-the-codespaces-browser-editor
-            const isRemote = vscode.env.remoteName !== 'undefined';
-            if (isRemote) {
-                dotnet = await acquireDotnetInstall(outputChannel);
-                await vscode.commands.executeCommand('dotnet.ensureDotnetDependencies', { command: dotnet, arguments: spawnedProxyArgs });
-            }
+            const dotnet = await acquireDotnetInstall(outputChannel);
+            await vscode.commands.executeCommand('dotnet.ensureDotnetDependencies', { command: dotnet, arguments: spawnedProxyArgs });
```

Comment:
@sfoslund could you fact check me here:

One thing to keep in mind, this attempts to "boot" the executable and detect if it crashes in a way that indicates there's a failure due to dependencies. Therefore, if this "ensureDotnetDependencies" piece stays running I think it may linger unless the proxy exits accordingly.

Reply:
Yes, that sounds right to me.

Reply:
@sfoslund Does the `ensureDotnetDependencies` stop the process it attempted to boot if it realizes that it did not fail due to missing dependencies? It seems like the answer is no?

Reply:
It just spawns a process with the arguments you provide as input to this command (see the code here: https://github.com/dotnet/vscode-dotnet-runtime/blob/master/vscode-dotnet-runtime-library/src/extension.ts#L150). You'd probably just want to run something simple that's guaranteed to return quickly (like `dotnet --info`) because it won't stop the process if it's long running.